### ISSUE TITLE
POL Unprofessionalism and A waste of time

### ISSUE THREAD

**Issue by M1CH-the-Unlikely** [packetflooder/POLLauncherCode#4]
Related to the subject of being "stolen", the launcher was indeed based off a base of the Altis launcher (from one of the original developers at POL), but was heavily modified, and left _unused by staff (MIT, as you stated)_, so technically it is not copyright-able to the team itself, and not applicable in this case.

If what you are saying is true, 
then we have the right to claim DMCA as much as possible, since you are essentially breaking the Github TOS in various ways (d.3 + c.3, read it) for both Altis (as you are infringing the copyright on his code) and us, and thus allowing us to take action in that case. :)

Not forgetting also Relived (which you've stole code / claimed open-source from, illlegally),

![screenshot_108](https://user-images.githubusercontent.com/20247483/33648335-5ea8d718-da27-11e7-8af0-d1ec264c308c.jpg)

![screenshot_107](https://user-images.githubusercontent.com/20247483/33648039-facbff00-da25-11e7-9cd1-7d37c054c841.jpg)

![screenshot_105](https://user-images.githubusercontent.com/20247483/33647884-328ebf64-da25-11e7-92bf-437bb755821e.jpg)

![screenshot_106](https://user-images.githubusercontent.com/20247483/33648018-e2eb36c6-da25-11e7-836d-855166d51364.jpg)

tldr; we should feel ashamed of ourselves, we're salty tryhards lol ;)

thank you~
#POLEXPOSED #eyeswideopenemoj #more unprofessional posts

**Comment by Bob693** [comment]
Lol jean

Your team has leaked my info and done several unprofessional actions in the last 24 hours.

You're wasting your time.

**Comment by Bob693** [comment]
Number 2 doesn't even apply here nor does 8.

First off the repo was public and posted by @xseano

And number 8 we didn't harass you. You harassed me and leaked my info.

**Comment by Bob693** [comment]
I'm glad you're joking about a serious matter. Just makes you guys look worse.

**Comment by Bob693** [comment]
You guys singlehandly ruined your entire projects reputation in a matter of hours! Congratulations on this achievement!
